### ISSUE TITLE
Catkin_make error

### ISSUE THREAD

**Issue by kvnptl** [RoboGnome/VS_Code_ROS#1]
Hi,

I am installing visual studio code with ROS on Windows 10 machine.

Error: 

> catkin_make : The term 'catkin_make' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

I am new to the VS code. So, can you please suggest how to solve this error?

**Comment by lzptr** [comment]
Hi, 

sorry for the late response. Somehow I wasn't notified about the issue and I'm not that often on github :)

The catkin_make command is part of the ROS build system. You would need to follow the ROS installation steps first.
I tested this repo with a Ubuntu system. 
You would first need to instlal ROS:
http://wiki.ros.org/Installation/Ubuntu

and then setup the workspace for the catkin build system:
http://wiki.ros.org/catkin/Tutorials/create_a_workspace

But it's been a while since I worked with ROS, so maybe I'm forgetting a thing or two. Hope this helps.
